ampersand-form-view.js

```diff
@@ -124,7 +126,7 @@ module.exports = View.extend({
     },
 
     remove: function () {
-        this.el.removeEventListener('submit', this.handleSubmit, false);
+        this.formEl.removeEventListener('submit', this.handleSubmit, false);
```

Comment:
This would throw an error if there's ever a case that `remove` is called before `render` is called. I'm not sure that would ever occur, but if it's a possibility, this line should be prefixed with `this.formEl && ....`